src/drive/web/modules/views/Recent/index.spec.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import { useQuery } from 'cozy-client'
+import { render, fireEvent } from '@testing-library/react'
+import { Router, hashHistory, Route } from 'react-router'
+
+import { setupStoreAndClient } from 'test/setup'
+import { generateFile } from 'test/generate'
+import AppLike from 'test/components/AppLike'
+
+import RecentViewWithProvider from './index'
+import { useFilesQueryWithPath } from './useFilesQueryWithPath'
+
+jest.mock('cozy-client/dist/hooks/useQuery', () => jest.fn())
+jest.mock('./useFilesQueryWithPath', () => ({
+  ...jest.requireActual('./useFilesQueryWithPath'),
+  useFilesQueryWithPath: jest.fn()
+}))
+
+describe('Recent View', () => {
+  const setup = ({
+    nbFiles = 5,
+    totalCount,
+    useQueryResultAttributes,
+    path = '/test',
+    dir_id = 'io.cozy.files.root-dir',
+    updated_at = '2020-05-14T10:33:31.365224+02:00'
+  } = {}) => {
+    const { store, client } = setupStoreAndClient({
+      selection: {
+        selected: {}
+      }
```

Comment:
I think you should wrap the selection slice into the `initialStoreState` attribute.